srcpkgs/vscode/REMOVE

```diff
@@ -1,8 +1,8 @@
 if [ "$UPDATE" = "no" ]; then
     case "$ACTION" in
         post)
-            if [ -d usr/share/code-oss ]; then
```

Comment:
Don't you also have to handle the removal of `/usr/share/code-oss` on this update?

Reply:
Good point! Will fix it.

Reply:
Fixed.